server/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
agreed.  I *think* this got added by pycharm somehow?  anyway, I did not notice it and it should be removed.